tools/extra/integrated/hssi/e100.cpp

```diff
@@ -1,5 +1,31 @@
+// Copyright(c) 2018, Intel Corporation
```

Comment:
If this was part of a prior release that went out last year, you could use 2017-2018.  Looks good.